plugins/transport-grpc/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@amplication/plugin-transport-grpc",
+  "version": "0.0.1",
+  "description": "set grpc as protocol for Amplication build",
```

Comment:
Please change the description to "add gRPC endpoints to your generated services"